core/src/test/java/com/homeaway/streamplatform/streamregistry/utils/EmbeddedKafkaStack.java

```diff
@@ -113,6 +113,7 @@ private Properties effectiveBrokerConfigFrom(Properties brokerConfig, ZooKeeperE
         effectiveConfig.put(KafkaConfig$.MODULE$.DeleteTopicEnableProp(), true);
         effectiveConfig.put(KafkaConfig$.MODULE$.LogCleanerDedupeBufferSizeProp(), 2 * 1024 * 1024L);
         effectiveConfig.put(KafkaConfig$.MODULE$.GroupMinSessionTimeoutMsProp(), 0);
+        effectiveConfig.put(KafkaConfig$.MODULE$.GroupInitialRebalanceDelayMsProp(), 0);
```

Comment:
A new config, group.initial.rebalance.delay.ms, was introduced. This config specifies the time, in milliseconds, that the GroupCoordinator will delay the initial consumer rebalance. The rebalance will be further delayed by the value of group.initial.rebalance.delay.ms as new members join the group, up to a maximum of max.poll.interval.ms. The default value for this is 3 seconds. During development and testing it might be desirable to set this to 0 in order to not delay test execution time.